addons/binding/org.openhab.binding.windcentrale/ESH-INF/i18n/windcentrale_nl.properties

```diff
@@ -0,0 +1,24 @@
+# binding
+binding.windcentrale.name = WindCentrale Binding
+binding.windcentrale.description = Binding voor WindCentrale reports.
+
+# thing types
+thing-type.windcentrale.mill.label = WindCentrale WindMolen
+thing-type.config.windcentrale.mill.millId.label = Wind Molen
+thing-type.config.windcentrale.mill.wd.label = Aantal Wind Delen
+thing-type.config.windcentrale.mill.wd.description = Aantal Wind delen in bezit
+thing-type.config.windcentrale.mill.refreshInterval.label = Ververs interval
+thing-type.config.windcentrale.mill.refreshInterval.description = ververs interval in seconden.
+
+# channel types
+channel-type.windcentrale.windSpeedType.label = Wind Snelheid (Bf)
+channel-type.windcentrale.windDirectionType.label = Wind Richting
+channel-type.windcentrale.powerKType.label = Productie molen
+channel-type.windcentrale.powerRelType.label = Productie Vermogen (%)
+channel-type.windcentrale.energyType.label = Vermogen
+
+# channels
+channel.windcentrale.powerAbsTot.label = Productie Vermogen (%)
+channel.windcentrale.powerAbsWd.label = Vermogen van de winddelen
+channel.windcentrale.kwhForecast.label = Verwachte Productie
+channel.windcentrale.runPercentage.label = Molen Productie percentage
```

Comment:
It would be better if the same capitalization style was used everywhere in the translations. So either:

* each sentence starts with a capital and uses lowercase for the rest of the sentence (excl. branded words)

or

* each word is capitalized

The same applies to the English translations.